src/VisualStudio/Razor/Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient.csproj

```diff
@@ -19,4 +19,8 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Workspaces\Core\Portable\Microsoft.CodeAnalysis.Workspaces.csproj" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@tmat I just noticed. This ItemGroup being removed is probably the crux of the issue actually given this is `Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient.csproj`

Reply:
That being said of course, I imagine we'd want to transition everything from `Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient.csproj` into `ExternalAccess.Razor` right?

Reply:
You are right. I forgot this hasn't been ExternalAccess-ized. Yes, we can actually remove this assembly and add ExternalAccess APIs to Workspaces. That would be much simpler. I'm gonna add these.